### PR TITLE
PTRENG-6489 - Add http request timeout and request compression Fluent…

### PR DESCRIPTION
**Pull request by benharosh**
* Add support for metrics and logs outbound payload compression, with `gzip_compression` FluentD param as part of `fluent-plugin-jfrog-sent-metrics` (new param) and `fluent-plugin-splunk-hec` (exposing existing configurable param) FluentD plugins
* Add support for configurable http request timeout, with `request_timeout` FluentD param as part of `fluent-plugin-jfrog-metrics` and `fluent-plugin-jfrog-sent-metrics` plugins
* Add support for a configurable `verify_ssl` FluentD param as part of `fluent-plugin-jfrog-metrics`
* FluentD sidecar version bumped to 4.9, to incorporate the above changes

### CODE REVIEW

File: fluent.conf.rt [hunk 1]
```diff
@@ -3,12 +3,14 @@
   @type jfrog_metrics
   @id metrics_http_jfrt
   tag jfrog.metrics.artifactory
-  interval 5s
+  execution_interval 60s
```
Comment (danielmkn) [inline fluent.conf.rt:6]: Is this correct?

**Review by danielmkn — APPROVED** [review]

File: fluent.conf.xray [hunk 2]
```diff
@@ -24,6 +24,7 @@
   metric_name_key metric_name
   metric_value_key value
   insecure_ssl "#{ENV['SPLUNK_INSECURE_SSL']}"
+  gzip_compression "#{ENV['SPLUNK_COMPRESS_DATA']}"
```
Comment (MahithaB) [inline fluent.conf.xray:27]: no execution interval and request timeout for an x-ray?

File: fluent.conf.rt (hunk 1 above)
Comment (benharosh) [reply fluent.conf.rt:6]: yes, it was an error before

File: fluent.conf.xray (hunk 2 above)
Comment (benharosh) [reply fluent.conf.xray:27]: Good point. I'll add them

**Review by MahithaB — APPROVED** [review]

**Pull request merged**